functions/studies/studies.py

```diff
@@ -7,14 +7,37 @@
 
 import requests
 import constant
+import logging
 
 #
 # @author Sylvain Bouzols <sylvain.bouzols at rte-france.com>
 #
 
+# Configure logging
+logging.basicConfig(level=logging.INFO)
```

Comment:
what is it for ? It seems like we are always indicating which level to use when logging something